scripts/mkdocs.sh

```diff
@@ -16,11 +16,13 @@ if [ ! -f docs/source/modules.rst ]
 
     fi;
 
-python -m m2r README.md changelog.md .github/CONTRIBUTING.md --overwrite
+python_path=$(which python)
```

Comment:
Thanks, looks good. Since `python` may be `python3`, could you check with `|| python3`? This assumes that we shall either have `python` or `python3` not both. 

Could you also add `bash scripts/mkdocs.sh` to [workflows](https://github.com/Nelson-Gon/pycite/.github/workflows) to ensure that the script works as expected? On testing, I got the following issues
* `sphinx-build` command not found. This may be due to issues with `pip` vs `apt install`.

Reply:
I have added Sphinx to requirements.txt as it is necessary to run `sphinx-build` command. I have also added the 'bash scripts/mkdocs.sh' command to workflow. Please review the changes.

Reply:
Fixed simple typo, thanks for your contribution.

Keep Building,

Nelson